lucene/core/src/test/org/apache/lucene/index/TestOrdinalMap.java

```diff
@@ -32,6 +28,10 @@
 import org.apache.lucene.util.RamUsageTester;
 import org.apache.lucene.util.TestUtil;
 
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.HashMap;
+
```

Comment:
Does (would) https://issues.apache.org/jira/browse/LUCENE-9564 enforce import ordering check?

Reply:
I think it would avoid this sort of change (I use IntelliJ which autoformats  `java` imports at the end).